packages/plugins/apollo-usage-report/package.json

```diff
@@ -41,7 +41,10 @@
     "graphql-yoga": "workspace:^"
   },
   "dependencies": {
+    "@apollo/server-gateway-interface": "^1.1.1",
     "@apollo/usage-reporting-protobuf": "^4.1.1",
+    "@apollo/utils.usagereporting": "^2.1.0",
+    "@graphql-tools/utils": "10.9.0-alpha-20250601230319-2f5f24f393b56915c1c913d3f057531f79991587",
```

Comment:
@EmrysMyrddin FYI: I just noticed that this PR was merged with this alpha version in `package.json`.